index.html

```diff
@@ -14,13 +14,14 @@
         <p id="killerName"></p>
```

Comment:
I see that the starting body tag is missing here, you have the end one below. Maybe is a good idea to pop that in?

Reply:
Good idea indeed!